docs/README.txt

```diff
@@ -3,7 +3,7 @@ TVM Documentations
 This folder contains the source of TVM documents
 
 - A hosted version of doc is at https://tvm.apache.org/docs
-- pip install sphinx>=1.5.5 sphinx-gallery sphinx_rtd_theme matplotlib Image recommonmark "Pillow<7" autodocsumm tlcpack-sphinx-addon
+- pip install "sphinx>=1.5.5" sphinx-gallery sphinx_rtd_theme matplotlib Image recommonmark "Pillow<7" "autodocsumm<0.2.0" tlcpack-sphinx-addon
```

Comment:
My only comment is a small one. When we pin, we need to document why, and under what conditions we can un-pin. I can't speak to sphinx or Pillow, but we should link to this and note we can remove the pin when the change is in upstream packages. https://github.com/Chilipp/autodocsumm/pull/42#issuecomment-818243851

Reply:
Good point, and that makes sense.  I will add that to the README.txt shortly.

There's probably also room for the [gen_requirements.py](https://github.com/apache/tvm/blob/main/python/gen_requirements.py) to be used more in the docs/examples in the future.  From what I can tell, the list of dependencies in it would be a perfect spot to centralize all dependency listings.  Right now, it looks like it is primarily used in the CI framework, but the `tvm/python/requirements/*.txt` could be referred anywhere that currently has an explicit listing of the required packages.